test/upgrades/RibbonThetaVault.ts

```diff
@@ -75,161 +55,63 @@ describe("RibbonThetaVault upgrade", () => {
-    strikeSelection:
-      "0x00000000000000000000000039d3799b8abefc3d05db5ba3b3b2770146475000",
-    premiumDiscount:
-      "0x00000000000000000000000000000000000000000000000000000000000000c8",
-    currentOtokenPremium:
-      "0x0000000000000000000000000000000000000000000000000003a62e9940d000",
-    lastStrikeOverrideRound:
-      "0x0000000000000000000000000000000000000000000000000000000000000000",
-    overriddenStrikePrice:
-      "0x0000000000000000000000000000000000000000000000000000000000000000",
-    auctionDuration:
-      "0x0000000000000000000000000000000000000000000000000000000000000e10",
-    optionAuctionID:
-      "0x0000000000000000000000000000000000000000000000000000000000000039",
-  });
-
-  checkIfStorageNotCorrupted("0x65a833afDc250D9d38f8CD9bC2B1E3132dB13B2F", {
-    totalSupply:
-      "0x00000000000000000000000000000000000000000000000000000000899fc95a",
-    name: "0x526962626f6e20425443205468657461205661756c740000000000000000002c", // Ribbon BTC Theta Vault
-    symbol:
-      "0x724254432d544845544100000000000000000000000000000000000000000014", // symbol - rBTC-THETA
-    vaultParams1:
-      "0x000000000000000000002260fac5e5542a773aa44fbcfedf7c193bc2c5990800",
-    vaultParams2:
-      "0x0000000000000000000003e82260fac5e5542a773aa44fbcfedf7c193bc2c599",
-    cap: "0x000000000000000000000000000000000000000000000000000000009502f900",
-    vaultState1:
-      "0x00000000000000000000000000772c200e0000000000000000008ae2222d0005",
-    vaultState2:
-      "0x000000000000000000000000010f28d40000000000000000000000000899be61",
-    optionState:
-      "0x00000000000000006156ef253f5f049ef271cdf18edab66e80e3209ecae5a228",
-    feeRecipient:
-      "0x000000000000000000000000daeada3d210d2f45874724beea03c7d4bbd41674",
-    keeper:
-      "0x000000000000000000000000a4290c9eae274c7a8fbc57a1e68adc3e95e7c67e",
-    performanceFee:
-      "0x0000000000000000000000000000000000000000000000000000000000989680",
-    managementFee:
-      "0x00000000000000000000000000000000000000000000000000000000000095d4",
-    optionsPremiumPricer:
-      "0x000000000000000000000000d8bb660a8fcaeadb7a7aef73e57a3a989065dacc",
-    strikeSelection:
-      "0x0000000000000000000000005e68b6f5c82fc5f3711541ca4a12e01b967fc641",
-    premiumDiscount:
-      "0x00000000000000000000000000000000000000000000000000000000000000c8",
-    currentOtokenPremium:
-      "0x00000000000000000000000000000000000000000000000000030419624c6800",
-    lastStrikeOverrideRound:
-      "0x0000000000000000000000000000000000000000000000000000000000000000",
-    overriddenStrikePrice:
-      "0x0000000000000000000000000000000000000000000000000000000000000000",
-    auctionDuration:
-      "0x0000000000000000000000000000000000000000000000000000000000000e10",
-    optionAuctionID:
-      "0x000000000000000000000000000000000000000000000000000000000000003a",
-  });
+  checkIfStorageNotCorrupted("0x25751853Eab4D0eB3652B5eB6ecB102A2789644B");
```

Comment:
1) Should this be the new address or the old address?

2) Could we define these addresses as like `RIBBON_THETA_VAULT_LOGIC_225751` or pull it from the deployments constants `constants/deployments.json` so we don't refine the same variable in multiple spots?